front-end/src/visitor/VisitorFunction.cpp

```diff
@@ -62,27 +63,64 @@ antlrcpp::Any StartVisitor::visitFunctionDefinition(MapleGrammarParser::Function
 antlrcpp::Any StartVisitor::visitTypeList(MapleGrammarParser::TypeListContext *ctx) {
     auto fParams = new vector<FunctionParam *>();
 
-    for (std::size_t i = 0; i != ctx->ID().size(); i++) {
-        const string &name = ctx->ID(i)->getText();
+    for (std::size_t i = 0; i != ctx->argumentTypeVar().size(); i++) {
```

Comment:
Prefer a more optimized iterator `for (auto&& item: ctx->argumentTypeVar()) {[...]}`

Reply:
Ok